src/sentry/web/frontend/organization_auth_settings.py

```diff
@@ -113,6 +117,8 @@ def handle_existing_provider(
         if request.method == "POST":
             op = request.POST.get("op")
             if op == "disable":
+                if provider.is_partner:
+                    return HttpResponse("Can't disable partner authentication provider", status=405)
```

Comment:
This prevents disabling the auth provider with a request, but will it also prevent someone who tries to edit the settings through a request (not the UI)?